src/express/cart-handler.js

```diff
@@ -1,11 +1,16 @@
 const { SHIFTClient } = require('@shiftcommerce/shift-node-api')
 const { getSessionExpiryTime } = require('../lib/session')
 
+const defaultCartIncludes = {
+  include: 'line_items.item.product,line_items.line_item_discounts,discount_summaries,customer_account,billing_address,shipping_address,shipping_method'
+}
+
 module.exports = {
   getCart: async (req, res) => {
+    req.query = Object.assign({}, req.query, defaultCartIncludes)
```

Comment:
Is this right? wouldn't anything in req.query that was in defaultCartIncludes be overwritten by defaultCartIncludes?

More consise might be using the spread operator
{...defaultCartIncludes, ...req.query}